### PR TITLE
fix for reward_item with spaces

### PR DESCRIPTION
**Pull request by marklonquist**
added simple test for admob
added strings.replace to change %20 to space in rawQuery variable

### CODE REVIEW

**Comment by marklonquist** [comment]
Fixes #2

File: admob/main.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func Verify(cbUrl *url.URL) (err error) {
 		return err
 	}
 
-	rawQuery := cbUrl.RawQuery
+	rawQuery := strings.Replace(cbUrl.RawQuery, "%20", " ", -1)
```
Comment (hiyali) [inline admob/main.go]: Please change this line with
```golang
rawQuery, err := url.QueryUnescape(cbUrl.RawQuery)
if err != nil {
    return err
}
```

**Review by hiyali — CHANGES_REQUESTED** [review]
And merge that commits to one commit, Thank you.

File: admob/main.go (hunk 1 above)
Comment (marklonquist) [reply admob/main.go]: That certainly makes sense. Why didn't I think of that :-) Hereby done!

**Comment by marklonquist** [comment]
> And merge that commits to one commit, Thank you.

working on this

**Pull request closed without merging**